integration_test/support/schemas.exs

```diff
@@ -46,13 +46,15 @@ defmodule Ecto.Integration.Post do
     field :intensities, {:map, :float}
     field :posted, :date
     has_many :comments, Ecto.Integration.Comment, on_delete: :delete_all, on_replace: :delete
+    has_many :ordered_comments, Ecto.Integration.Comment, preload_order: [asc: :text]
```

Comment:
I would use it like:
```suggestion
    has_many :ordered_comments, Ecto.Integration.Comment, preload_order: [:text]
```
because `:text` should work as `[asc: :text]`, so we would test 2 things (short notation + ascending order) at same time.

Maybe we want to strictly test all 3 cases i.e. `[:text]`, `[asc: :text]` and `[desc: :text]`, but at least 2 of them (`[:text]` and `[desc: :text]`) should be tested.

Reply:
@Eiji7 I added support for the short notation. WDYT?